erpnext/public/js/shopping_cart.js

```diff
@@ -36,7 +36,6 @@ $.extend(shopping_cart, {
 	},
 
 	update_cart: function(opts) {
-		var full_name = frappe.session.user_fullname;
```

Comment:
I 've noticed frappe.session.user_fullname was empty, so i just revert to the last version which worked but indeed use global. Thanks for review.